app/models/occupation.rb

```diff
@@ -4,6 +4,30 @@ class Occupation < ApplicationRecord
   has_many :occupation_standards
   has_many :industries, through: :occupation_standards
 
+  class << self
+    def search(q)
```

Comment:
This query looks a lot more complicated than i was expecting for what it's doing. Good for now, but I'd love to talk through what you were seeing that made this necessary during the tech check-in

Reply:
Searching the array field seemed to be not trivial, unless there's a simpler way to do it that I couldn't find. I don't think I've ever run into needing to search array fields before. I can take another look, or if you have a better solution implemented in workhands, let me know!

Reply:
Ah! The array field. That explains it.

Yeah - if we need to build on this a lot more, it might be just simpler to create a separate `Alias` table with this then. I don't know that we'll need to do a ton more to this search, though, so let's tackle that when necessary.